packages/status-js/src/protos/chat-message_pb.ts

```diff
@@ -0,0 +1,534 @@
+    proto3.util.initPartial(data, this);
+  }
+
+  static readonly runtime = proto3;
+  static readonly typeName = "DeleteMessage";
+  static readonly fields: FieldList = proto3.util.newFieldList(() => [
+    { no: 1, name: "clock", kind: "scalar", T: 4 /* ScalarType.UINT64 */ },
+    { no: 2, name: "chat_id", kind: "scalar", T: 9 /* ScalarType.STRING */ },
+    { no: 3, name: "message_id", kind: "scalar", T: 9 /* ScalarType.STRING */ },
+    { no: 4, name: "grant", kind: "scalar", T: 12 /* ScalarType.BYTES */ },
+    { no: 5, name: "message_type", kind: "enum", T: proto3.getEnumType(MessageType) },
+  ]);
+
+  static fromBinary(bytes: Uint8Array, options?: Partial<BinaryReadOptions>): DeleteMessage {
+    return new DeleteMessage().fromBinary(bytes, options);
+  }
+
+  static fromJson(jsonValue: JsonValue, options?: Partial<JsonReadOptions>): DeleteMessage {
+    return new DeleteMessage().fromJson(jsonValue, options);
+  }
+
+  static fromJsonString(jsonString: string, options?: Partial<JsonReadOptions>): DeleteMessage {
+    return new DeleteMessage().fromJsonString(jsonString, options);
+  }
+
+  static equals(a: DeleteMessage | PlainMessage<DeleteMessage> | undefined, b: DeleteMessage | PlainMessage<DeleteMessage> | undefined): boolean {
+    return proto3.util.equals(DeleteMessage, a, b);
+  }
+}
+
+/**
+ * @generated from message ChatMessage
+ */
+export class ChatMessage extends Message<ChatMessage> {
+  /**
+   * Lamport timestamp of the chat message
+   *
+   * @generated from field: uint64 clock = 1;
+   */
+  clock = protoInt64.zero;
+
+  /**
+   * Unix timestamps in milliseconds, currently not used as we use whisper as more reliable, but here
+   * so that we don't rely on it
+   *
+   * @generated from field: uint64 timestamp = 2;
+   */
+  timestamp = protoInt64.zero;
+
+  /**
+   * Text of the message
+   *
+   * @generated from field: string text = 3;
+   */
+  text = "";
+
+  /**
+   * Id of the message that we are replying to
+   *
+   * @generated from field: string response_to = 4;
+   */
+  responseTo = "";
+
+  /**
+   * Ens name of the sender
+   *
+   * @generated from field: string ens_name = 5;
+   */
+  ensName = "";
+
+  /**
+   * Chat id, this field is symmetric for public-chats and private group chats,
+   * but asymmetric in case of one-to-ones, as the sender will use the chat-id
+   * of the received, while the receiver will use the chat-id of the sender.
+   * Probably should be the concatenation of sender-pk & receiver-pk in alphabetical order
+   *
+   * @generated from field: string chat_id = 6;
+   */
+  chatId = "";
+
+  /**
+   * The type of message (public/one-to-one/private-group-chat)
+   *
+   * @generated from field: MessageType message_type = 7;
+   */
+  messageType = MessageType.UNKNOWN_MESSAGE_TYPE;
+
+  /**
+   * The type of the content of the message
+   *
+   * @generated from field: ChatMessage.ContentType content_type = 8;
+   */
+  contentType = ChatMessage_ContentType.UNKNOWN_CONTENT_TYPE;
```

Comment:
⚠️ Uses enum values.

See ["Why not use string unions for Protobuf enumerations instead of TypeScript enum?"](https://github.com/bufbuild/protobuf-es/blob/main/docs/faq.md#why-not-use-string-unions-for-protobuf-enumerations-instead-of-typescript-enum).